<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win</class>
 <widget class="QWidget" name="Win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <widget class="QPushButton" name="btnSend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>570</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearInput">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>570</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>清空输入列表</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>981</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>981</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="slbIP">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>570</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="slbPort">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>570</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIP">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePort">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>570</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDial">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>570</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="slbStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStatus">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>尚未连接到服务器</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
